dbt/seeds/ccao/schema.yml

```diff
@@ -34,6 +34,24 @@ seeds:
       - name: max_age
         description: Double of maximum age for 200-class property codes
 
+  - name: ccao.loa
+    description: '{{ doc("seed_loa") }}'
+    config:
+      column_types:
+        year: string
+        class_code: string
+
+    columns:
+      - name: year
+        description: '{{ doc("shared_column_year") }}'
+      - name: class_code
+        description: '{{ doc("shared_column_class") }}'
```

Comment:
**[Thought, non-blocking]** I wonder if we should file an issue to test this view to make sure that the class codes are all in `ccao.class_dict`?

Reply:
There are a _bunch_ of classes in the LoA table that unfortunately aren't in the class dict. Most of them don't have any instances. I'll defer fixing/adding tests until #376.